lib/cc/engine/analyzable_files.rb

```diff
@@ -27,13 +26,17 @@ def fetch_files(paths)
       end
 
       def build_files_with_inclusions(inclusions)
-        fetch_files(inclusions)
+        if inclusions == ["./"]
```

Comment:
I think that's correct *most of the time*. It just fails when you get passed `./` (which is default) or the user explicitly passes in anything beginning with `./`.